<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phuong Nghi_K234111403</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>721</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Chương Trình Tính Khấu Hao TSCD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Giá Mua Mới:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGiaMuaMoi">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhiVanChuyen">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Phí vận chuyển:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Phí lắp đặt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhiLapDat">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSoNamSuDungDuKien">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Số năm sử dụng dự kiến:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTinhKhauHao">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>1. Tính Khấu Hao</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNguyenGiaTSCD">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nguyên giá TSCD:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMucTrichKhauHaoNam">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mức trích khấu hao năm:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMucTrichKhauHaoThang">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>360</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mức trích khấu hao tháng:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonXemChiTietKhauHao">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>2. Xem chi tiết Khấu Hao</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>551</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Năm</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sau khấu hao còn =&gt;</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
